apps/preview/src/AppConsole.tsx

```diff
@@ -0,0 +1,51 @@
+import { useEffect } from "react";
+
+interface CustomConsole {
+  _log: (...keys: unknown[]) => void;
+  _warn: (...keys: unknown[]) => void;
+  _error: (...keys: unknown[]) => void;
+}
+
+const AppConsole = (): null => {
```

Comment:
Make this into a hook instead of a component.

Reply:
Is this fine to create this hook only for preview project scope, not in framework or runtime. ?

Reply:
Yes